### ISSUE TITLE
Please dispose all AnimationControllers

### ISSUE THREAD

**Issue by Eerey** [joo6077/cool_dropdown#14]
`drop_down_body.dart`

My proposal:
```dart
  @override
  void dispose() {
    _animationController.dispose();
    _triangleController.dispose();
    _scrollController.dispose();
    for (final anim in _DCController) {
      anim.dispose();
    }
    super.dispose();
  }
```

**Comment by Eerey** [comment]
Also, there are two list of animation controllers that are basically the same:

  final List<AnimationController> _DCController = [];
  final List<AnimationController> _paddingController = [];
  
  
  It's hard to follow the logic if both lists have the same objects.

**Comment by joo6077** [comment]
Yeah I need to refactory this old code. I have been planned to refactory overall of this library soon.

Thank you for letting me know tho!
